### PR TITLE
Minor editorial changes from RoP Audit Meeting

### PR DESCRIPTION
**Pull request by tfrauenstein**

### CODE REVIEW

**Comment by fmueller** [comment]
👍

File: general-guidelines/GeneralGuidelines.md [hunk 1]
```diff
@@ -60,18 +58,18 @@ Services also have to support an
...
+In addition to the API as OpenAPI Reference Definition, it’s good practice to provide 
+an API User Manual documentation to improve client developer experience, especially of engineers that 
+are less experienced in using this API. A helpful API User Manual typically describes 
 the following API aspects:
 
 - API’s scope, purpose and use cases
-- concrete usage examples 
-- architecture context - including figures and sequence flows
-- edge cases and possible error situations
+- concrete usage examples of API usage
```
Comment (ePaul) [inline general-guidelines/GeneralGuidelines.md]: 2× usage?

Maybe better:
> concrete examples of API usage

Comment (tfrauenstein) [inline general-guidelines/GeneralGuidelines.md]: done

**Comment by tfrauenstein** [comment]
👍

**Comment by ePaul** [comment]
:+1:

**Pull request merged**